### DISCUSSION TITLE
Support more richtext format when copy-and-paste, like google docs and slack

### DISCUSSION THREAD

**Discussion by Stupidism** [payloadcms/payload#1125] [Feature Requests & Ideas]
### Background

Our editor found bold and italic information missing when paste into payload from google docs. 
So I had a deeper dig found the richtext format from some platforms are not supported:

- [x] asana
- [x] contentful
- [ ] google docs
- [ ] slack

### From google docs

<img width="1147" alt="image" src="https://user-images.githubusercontent.com/5801015/190089518-e0177f1e-f8a6-4609-88cf-04aa0d904440.png">

<img width="781" alt="image" src="https://user-images.githubusercontent.com/5801015/190089611-c9a3dc1e-64fb-4819-972d-cdc9cb57e13e.png">

### From slack

<img width="607" alt="image" src="https://user-images.githubusercontent.com/5801015/190088268-ae768020-8898-4a71-a6e7-48a97e508649.png">

Only italic is copied 
<img width="729" alt="image" src="https://user-images.githubusercontent.com/5801015/190088419-b16115fe-191f-4a30-9b31-b6860383db97.png">

**Comment by EasonSoong** [comment]
Paste link from contentful couldn't work either. 
![image](https://user-images.githubusercontent.com/57319837/192670549-3c462540-509a-4409-b7b4-42306c585ea0.png)
![image](https://user-images.githubusercontent.com/57319837/192670634-aea51ada-0012-4a93-a43d-39ed0cb1fd36.png)

**Comment by EasonSoong** [comment]
![image](https://user-images.githubusercontent.com/57319837/192670467-b0cd00b1-b186-4221-809c-65412a67463f.png)

I notice if `data.types.includes('application/x-slate-fragment'` , the code won't handle the pasted data deserialize, and pass the data to editor.insertData will causes this issue.
